websites/F/Folding@home/iframe.ts

```diff
@@ -0,0 +1,14 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+  const points = document.querySelector("#box-points-team > a"),
+  progress = document.querySelector("#slot-0 > div.pu > div.pu-bar.ui-progressbar.ui-widget.ui-widget-content.ui-corner-all > div");
+  if (progress && points !== undefined) {
+  iframe.send({
+    iframe_d: {
```

Comment:
camelCase for `iframe_d`

Reply:
just setting a better name in general would be good